libr/io/undo.c

```diff
@@ -127,7 +127,7 @@ R_API RList *r_io_sundo_list(RIO *io, int mode) {
 
 	idx = io->undo.idx;
 	start = (idx - undos + R_IO_UNDOS) % R_IO_UNDOS;
-	end = (idx + redos + 1) % R_IO_UNDOS;
+	end = (idx + redos + 1 - 1) % R_IO_UNDOS; // +1 slot for current position, -1 due to inclusive end
```

Comment:
+1-1? just drop that

Reply:
I intentionally left the intermediate steps to make it clear what components are taken into account. Might be better as 'idx + (1 + redos) -1' as current element is before redo.